### DISCUSSION TITLE
Do not add header on create definition

### DISCUSSION THREAD

**Discussion by vitviper** [microsoft/vscode-cpptools#11501] [General]
Is there a setting to indicate I do not what the header file added to my c/cpp code every time I use "quick fix->create definition"? It always adds the header file without a path, which is never correct in my case. I would prefer it just not add the header.

**Comment by bobbrow** [comment]
Hi @vitviper, we are adding a setting for this in 1.18 (see #11271).  The work is complete but we haven't released it yet.  We hope to have a pre-release available next week.

**Reply by vitviper** [reply]
@bobbrow Thank you! I searched and searched and had not found anything on this.
